src/main/java/uk/gov/ons/ctp/response/action/service/ActionService.java

```diff
@@ -295,4 +296,20 @@ public Action updateAction(final Action action) {
     }
     return existingAction;
   }
+
+  public void rerunAction(final Action action) {
+    log.with("action_Id", action.getId()).debug("Rerunning Aborted Action");
```

Comment:
`getId()` returns a UUID, which might not log out very nicely. We often call `.toString()` to make sure we can see the UUID in the logs.